common/tls/src/main/java/io/helidon/common/tls/TlsConfigBlueprint.java

```diff
@@ -28,12 +28,16 @@
 
 import io.helidon.builder.api.Prototype;
 import io.helidon.common.pki.Keys;
+import io.helidon.common.tls.spi.TlsManagerProvider;
 import io.helidon.config.metadata.Configured;
 import io.helidon.config.metadata.ConfiguredOption;
 
 @Prototype.Blueprint(decorator = TlsConfigDecorator.class)
 @Configured
 interface TlsConfigBlueprint extends Prototype.Factory<Tls> {
+    /**
+     * The default protocol is set to {@code TLS}.
+     */
     String DEFAULT_PROTOCOL = "TLS";
```

Comment:
you can use `{@value}`  - this way even if the constant changes, the javadoc is still correct.